### ISSUE TITLE
The easiest way to integrate with serverless framework

### ISSUE THREAD

**Issue by idumchykov** [alexcasalboni/aws-lambda-power-tuning#159]
Is there the easiest way to convert sam to serverless, so it would be deployed using `sls deploy` command?

**Comment by alexcasalboni** [comment]
Hi @idumchykov 👋  thanks for asking!

You can include the SAR app in your `serverless.yml` file.

The app resource (`AWS::Serverless::Application `) will look like this: https://github.com/alexcasalboni/aws-lambda-power-tuning/blob/master/scripts/deploy-sar-app.yml#L6

And here's a quick guide to make it work with the Serverless Framework: https://medium.com/theburningmonk-com/how-to-include-serverless-repository-apps-in-serverless-yml-6d8233c5d684

Basically, you need to add two things:

1. The `Transform: AWS::Serverless-2016-10-31` transformation (AWS SAM)
2. The `AWS::Serverless::Application` resource (AWS SAR app)

I'm closing this issue for now, please feel free to reopen it in case you encounter any issues.

**Comment by idumchykov** [comment]
Thanks @alexcasalboni!
